rholang/README.md

```diff
@@ -25,57 +27,28 @@ The language is still in the early stages of development, but for those who are
     * JFlex - install using apt 
     * BNFC - must be built from [git](https://github.com/BNFC/bnfc) b0252e5f666ed67a65b6e986748eccbfe802bc17 or later
     * Scala
-4. Run `sbt bnfc:generate` to generate the lexer/parser. Re-run whenever you modify the grammar
-5. Run `sbt compile` to compile classes
-6. Run `sbt assembly` to build a stand-alone `.jar` file
+4. Run `sbt rholang/bnfc:generate` to generate the lexer/parser. Re-run whenever you modify the grammar
+5. Run `sbt rholang/compile` to compile classes
+6. Run `sbt rholangCLI/assembly` to build a stand-alone `.jar` file
 
 ### Command-line usage
 
 ```
-$ ./rho2rbl examples/token.rho
-compiled examples/token.rho to examples/token.rbl
-```
-which is short for:
-```
-$ java -jar target/scala-2.11/rholang-assembly-0.1-SNAPSHOT.jar examples/token.rho 
-compiled examples/token.rho to examples/token.rbl
+$ java -jar rholang-cli/target/scala-2.12/rholangCLI-assembly-0.1.0-SNAPSHOT.jar rholang/examples/token.rho
```

Comment:
`token.rho` (along with most of the other files in the `examples` folder) have not yet been translated to 0.2 syntax yet. We should definitely reference a file which actually works with the interpreter. Whether that means translating `token.rho` and leaving this line in the readme as is, or changing this line to use a working example (e.g. `stdout.rho`) I'm not sure.

Reply:
Feel free to name an example, and I'll change it.

Reply:
Maybe for now use `rholang/tests/mercury-tut/contract_test.rho` since we don't have ready translated examples? Or as part of this PR we create 0.2 examples.